Bit-Quill/amazon-timestream-tools
# [AT-1476] Amazon timestream in PHP example app

## Summary
Create the started application Amazon Timestream with PHP

## Description
Provide a starter app for PHP to use the Amazon Timestream application and the installation guide.

## Related Issue
[awslabs#42](https://github.com/awslabs/amazon-timestream-tools/issues/42)

## Tests performed / created
### Manual verification:
1. Install PHP based on the guide
2. Install Amazone SDK with Composer
3. Execute the application by `main.php`
4. Verify that database, table and records has been created
5. Verify query results
6. Verify table and database successfully deleted

## Additional Reviewers
@alexeytemnikov
